Client/Assets/scrape_plugins.py

```diff
@@ -26,7 +26,7 @@ def main():
             shutil.move(downloadedFile, os.path.join(directory, file))
 
 def getLocaleList():
-    response = requests.get('https://l10n.mozilla-community.org/~flod/webstatus/api/?product=firefox-ios&txt')
+    response = requests.get('https://hg.mozilla.org/releases/mozilla-aurora/raw-file/default/browser/locales/all-locales')
```

Comment:
Pulled that URL from an email as mentioned in the bug. I'll flip this to point to Android.